frontend/packages/webterminal-plugin/integration-tests/features/customization/customization-of-web-terminal.feature

```diff
@@ -0,0 +1,83 @@
+              And user clicks on Save button to save web terminal settings
+             Then user should see meesage success alert on save
+
+
+        @regression @manual
+        Scenario: When web terminal timeout value is changed: WT-02-TC03
+            Given user has updated timeout value in Web Terminal Congiguration page
+              And user decided to save timeout value even after operator restarts or update
+              And user is at namespace "openshift-operators"
+             When user searches "DevWorkspaceTemplates" in Search page
+              And user selects "web-terminal-exec" resource
+              And user switches to the YAML tab
+             Then user should see new timeout value in "WEB_TERMINAL_IDLE_TIMEOUT" evnironment variable
+              And user should see "web-terminal.redhat.com/unmanaged-state: 'true'" got added in "metadata.annotations"
+
+        
+        @regression @manual
+        Scenario: When web terminal image value is changed: WT-02-TC04
+            Given user has updated image value in Web Terminal Congiguration page
+              And user decided to save image value even after operator restarts or update
+              And user is at namespace "openshift-operators"
+             When user searches "DevWorkspaceTemplates" in Search page
+              And user selects "web-terminal-tooling" resource
+              And user switches to the YAML tab
+             Then user should see new image value in 'spec.components' for resource name "web-terminal-tooling" in "container.image" value
+              And user should see "web-terminal.redhat.com/unmanaged-state: 'true'" got added in "metadata.annotations"
+
+
+        @regression
+        Scenario: When user selects not to save data after operator restarts or update: WT-02-TC05
+            Given user is at Web Terminal Configuration page
+             When user unchecks checkboxes to save data even after operator restarts or update
+              And user clicks on Save button to save web terminal settings
+             Then user should see meesage success alert on save
+
+
+        @regression @manual
+        Scenario: When user decided not to save timeout value: WT-02-TC06
+            Given user has updated timeout value in Web Terminal Congiguration page
+              And user decided to save timeout value even after operator restarts or update
+              And user is at namespace "openshift-operators"
+             When user decides not to save timeout value by unchecking the checkbox to save data even after operator restarts or update in Web Terminal Configuration page
+              And user searches "DevWorkspaceTemplates" in Search page
+              And user selects "web-terminal-exec" resource
+              And user switches to the YAML tab
+             Then user should see "web-terminal.redhat.com/unmanaged-state: 'false'" got added in "metadata.annotations"
+
+        
+        @regression @manual
```

Comment:
Please implement this test. If you do have not the time on Tuesday, please keep it and create a follow-up story.